src/MatrixAlgebraKit.jl

```diff
@@ -19,6 +19,9 @@ export eigh_full, eigh_vals, eigh_trunc
 export eigh_full!, eigh_vals!, eigh_trunc!
 export eig_full, eig_vals, eig_trunc
 export eig_full!, eig_vals!, eig_trunc!
+export gen_eig, gen_eig!
+export gen_eig_full, gen_eig_vals
+export gen_eig_full!, gen_eig_vals!
```

Comment:
Just because I don't know this: is there a distinction between hermitian/nonhermitian versions, just like the `eig`-`eigh` ?

Reply:
No, there's no specialized LAPACK support for *generalized* eigenproblems for symmetric or Hermitian matrices, only the general (stupid naming system...) case is supported.